polaris/_mixins.py

```diff
@@ -4,7 +4,9 @@
 from loguru import logger
 from pydantic import BaseModel, PrivateAttr, computed_field
 
-from polaris.utils.errors import PolarisChecksumError
+
+class PolarisChecksumError(ValueError):
```

Comment:
Because we are using the `FormattingMixin` in the errors.py file but also using the `PolarisChecksumError` in this file, we run into issues with circular imports. 

Having `PolarisChecksumError` defined here doesn't feel correct, I've just done it to get past errors. We could solve this by creating a `mixins` folder and defining each mixin in an individual file. What do we think of that?

Reply:
I've converted the PR to a draft until we decide what to do here.

Reply:
> We could solve this by creating a mixins folder and defining each mixin in an individual file

That works!

An alternative is to move the import of the `PolarisChecksumError` inside of `FormattingMixin.format()`.